### DISCUSSION TITLE
Deleting images from Media Manager

### DISCUSSION THREAD

**Discussion by PH5HP** [e107inc/e107#5229] [General]
Hi,

I was unable to delete an image from the Media Manager when I was in Grid View. Selecting the image and choosing DELETE at the bottom of the page and click on GO did not work. Also directly clicking on the RED CROSS button and confirming after that did not work.

I discovered that if I changed the view to List View both ways to DELETE the image do work. Is this a bug in Grid View. Anyone who can confirm my findings?

Kind regards,
Henk - ph5hp.nl - https://fileserver.ph5hp.nl (if my main computer is online)

**Comment by Jimmi08** [comment]
They say it should be fixed:

https://github.com/e107inc/e107/issues/5177

**Reply by PH5HP** [reply]
Hi,

Tnx. I suppose it will be fixed in the next release?

Kind regards,
Henk - ph5hp.nl - https://fileserver.ph5hp.nl/ (if my main computer is online)
